src/main/java/org/fcrepo/client/TransactionBuilder.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree.
+ */
+package org.fcrepo.client;
+
+import java.net.URI;
+
+import org.apache.http.client.methods.HttpRequestBase;
+import org.apache.http.util.Args;
+
+/**
+ * Creates different RequestBuilders for interacting with the Fedora Transaction API.
+ *
+ * @author mikejritter
+ */
+public class TransactionBuilder {
+
+    public static final String TRANSACTION_ENDPOINT = "fcr:tx";
+
+    private final URI uri;
+    private final FcrepoClient client;
+
+    /**
+     * Instantiate builder. Throws an IllegalArgumentException if either the uri or client are null.
+     *
+     * @param uri    uri of the resource this request is being made to
+     * @param client the client
+     */
+    public TransactionBuilder(final URI uri, final FcrepoClient client) {
+        Args.notNull(uri, "uri");
+        Args.notNull(client, "client");
+
+        this.uri = uri;
+        this.client = client;
+    }
+
+    /**
+     * Create a RequestBuilder to start a transaction
+     *
+     * @return the RequestBuilder
+     */
+    public RequestBuilder start() {
+        final URI target;
+        final var asString = uri.toString();
+        if (asString.endsWith(TRANSACTION_ENDPOINT)) {
```

Comment:
Might be good to add some validation of the other methods to verify that the uri is a txUri, maybe throwing an illegalargumentexception if not

Reply:
Yea I was trying to think of a better way to handle this than how it currently is. Part of me wants to add a `TransactionUri` class which is returned when calling `FcrepoResponse::getTransactionUri` and use that wherever we need the transaction id.

Reply:
Oh I forgot how the changes I made looked, maybe a `TransactionUri` class wouldn't work too well. I'll get some validation put in for the other methods.